tests/storage/zfsvol/test_zfsvol_sr.py

```diff
@@ -12,14 +12,17 @@
 
 pytestmark = pytest.mark.usefixtures("host_at_least_8_3")
 
+
+@pytest.mark.usefixtures("host_with_zfsvol")
```

Comment:
This is not needed at the moment since you also added the fixture to the only test in the class

Reply:
Ok. I thought that to use host_with_zfsvol as parameter I needed to also specify that it is a feature. I will remove it.